Tests/SnapshotTests.swift

```diff
@@ -0,0 +1,66 @@
+// Created by Cal Stephens on 12/8/21.
+// Copyright © 2021 Airbnb Inc. All rights reserved.
+
+@testable import Lottie
+import SnapshotTesting
+import XCTest
+
+// MARK: - SnapshotTests
+
+class SnapshotTests: XCTestCase {
+
+  /// Snapshots all of the sample animation JSON files visible to this test target
+  func testLottieSnapshots() {
+    #if !os(iOS)
+    // We only run snapshot tests on iOS, since running snapshot tests
+    // for macOS and tvOS would triple the number of snapshot images
+    // we have to check in to the repo.
+    XCTFail("Snapshot tests should only be ran on iOS")
+    #endif
+
+    for sampleAnimationURL in Bundle.module.urls(forResourcesWithExtension: "json", subdirectory: nil)! {
+      let sampleAnimationName = sampleAnimationURL.lastPathComponent.replacingOccurrences(of: ".json", with: "")
+
+      guard let animation = Animation.named(sampleAnimationName, bundle: .module) else {
+        continue
```

Comment:
If we fail to decode the image should we throw an error? We wouldn't want this to fail silently I'd imagine

Reply:
Great point, I fixed this in #1433 (https://github.com/airbnb/lottie-ios/pull/1433/files#diff-c63a7ac51a7f84e89040a1f0e17375e14f26cc41720b2e3444385d8c7416dce0R25)